homeassistant/components/sensor/travel_time.py

```diff
@@ -0,0 +1,114 @@
+"""
+Support for RFXtrx sensors.
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/sensor.travel_time/
```

Comment:
Since this is service provided by the google maps APIs wouldn't it be better to have the full naming in  the case we get more sensors using the same set of APIs?
In this case I would suggest ```google_maps_travel_time```.
Continuing on the same naming scheme the suggested google calendar sensor could then be called ```google_apps_calendar``` since it's part of Google Apps set of APIs